cli/src/init.rs

```diff
@@ -566,7 +568,8 @@ pub fn generate_grammar_files(
                     let contents = fs::read_to_string(path)?;
                     let old = "add_custom_target(test";
                     if contents.contains(old) {
-                        write_file(path, contents.replace(old, "add_custom_target(ts-test"))
+                        write_file(path, contents.replace(old, "add_custom_target(ts-test"))?;
+                        Ok(())
                     } else {
                         Ok(())
                     }
```

Comment:
since we can't return the value of write_file here now, remove the else and hoist the `Ok` outside the block